### PR TITLE
Fix renamed module in a renamed package

### PR DESCRIPTION
**Pull request by stefanpenner**
- [x] confirm
- [x] test?

### CODE REVIEW

File: test-packages/macro-tests/tests/integration/components/funky-sample-addon-interopt-test.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { module, test } from 'qunit';
```
Comment (stefanpenner) [inline test-packages/macro-tests/tests/integration/components/funky-sample-addon-interopt-test.js]: @ef4  relevant test

File: test-packages/@embroider/funky-sample-addon/tests/unit/dummy-app-test.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import { module, test } from 'qunit';
```
Comment (stefanpenner) [inline test-packages/@embroider/funky-sample-addon/tests/unit/dummy-app-test.js]: @ef4  relevant test

File: test-packages/@embroider/funky-sample-addon/index.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+const funnel = require('broccoli-funnel');
+const mergeTrees = require('broccoli-merge-trees');
+const path = require('path');
+
+process.env.EMBER_CLI_IGNORE_ADDON_NAME_MISMATCH = true;
+module.exports = {
+  name: 'funky-sample-addon',
+
+  treeForAddon() {
+    const addonTree = this._super.treeForAddon.apply(this, arguments),
```
Comment (stefanpenner) [inline test-packages/@embroider/funky-sample-addon/index.js]: @ef4  relevant funky build behavior

File: test-packages/@embroider/funky-sample-addon/fake-other-module/index.js [hunk 4]
```diff
@@ -0,0 +1,4 @@
+export default {
```
Comment (stefanpenner) [inline test-packages/@embroider/funky-sample-addon/fake-other-module/index.js]: mimic to be compile programmatically by babel module

File: test-packages/@embroider/funky-sample-addon/addon/index.js [hunk 5]
```diff
@@ -0,0 +1,13 @@
+import FakeOther from 'fake-module';
```
Comment (stefanpenner) [inline test-packages/@embroider/funky-sample-addon/addon/index.js]: @ef4  import ad-hoc build by babel in tree module and use it.

**Comment by stefanpenner** [comment]
looks like we have a test failure, will look shortly.

**Comment by stefanpenner** [comment]
will investigate the failure in the AM.

**Comment by stefanpenner** [comment]
CI is happy!

**Comment by ef4** [comment]
Thanks, looking good.

**Pull request merged**